packages/backend/src/services/UnfurlService/UnfurlService.ts

```diff
@@ -392,6 +396,7 @@ export class UnfurlService extends BaseService {
         lightdashPage?: LightdashPage;
         chartType?: string;
         organizationUuid?: string;
+        authUserUuid: string;
```

Comment:
this saveScreenshot can come from a scheduled delivery OR a dashboard export on the UI